cookbook/integration_llama-index_milvus-lite.ipynb

```diff
@@ -0,0 +1,235 @@
+{
```

Comment:
<p>Nit,</p><p> </p><p>It uses <s>a </s>Milvus Lite to store</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/langfuse/langfuse-docs/pull/641/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>